.credo.exs

```diff
@@ -0,0 +1,13 @@
+# config/.credo.exs
+%{
+  configs: [
+    %{
+      name: "default",
+      checks: [
+        {Credo.Check.Readability.LargeNumbers, only_greater_than: 86400},
+        {Credo.Check.Readability.ParenthesesOnZeroArityDefs, parens: true},
+        {Credo.Check.Design.TagTODO, false}
```

Comment:
Sorry to add on yet another thing... I was on a call with Frank this morning and he mentioned how he wants us to utilize specs a bit more and to add {Credo.Check.Readability.Specs, tags: []}, to our credo. I don't know how big of a priority it is, but I realized I should mention it here

Reply:
Noted. Thanks!